geomstats/geometry/spd_matrices.py

```diff
@@ -348,11 +371,16 @@ def logm(cls, mat):
         """
         n = mat.shape[-1]
         dim_3_mat = gs.reshape(mat, [-1, n, n])
-        logm = cls.apply_func_to_eigvals(
+        logm = SymmetricMatrices.apply_func_to_eigvals(
             dim_3_mat, gs.log, check_positive=True)
         logm = gs.reshape(logm, mat.shape)
         return logm
 
+    expm = SymmetricMatrices.__dict__['expm']
```

Comment:
What is this? Why not just `expm = SymmetricMatrices.expm`?

Reply:
Try it if you want :stuck_out_tongue_closed_eyes: 

SymmetricMatrices.expm is a class method so what you suggest works, but it does not for `from_vector` and `to_vector` that are static,  because the first argument `self=SPD(n)` is not dealt with.